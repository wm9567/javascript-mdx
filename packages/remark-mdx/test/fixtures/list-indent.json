{
  "type": "root",
  "children": [
    {
      "type": "paragraph",
      "children": [
        {
          "type": "text",
          "value": "As we don’t support indented code, lists can be indented some more.",
          "position": {
            "start": {
              "line": 1,
              "column": 1,
              "offset": 0
            },
            "end": {
              "line": 1,
              "column": 68,
              "offset": 67
            }
          }
        }
      ],
      "position": {
        "start": {
          "line": 1,
          "column": 1,
          "offset": 0
        },
        "end": {
          "line": 1,
          "column": 68,
          "offset": 67
        }
      }
    },
    {
      "type": "list",
      "ordered": false,
      "start": null,
      "spread": false,
      "children": [
        {
          "type": "listItem",
          "spread": false,
          "checked": null,
          "children": [
            {
              "type": "paragraph",
              "children": [
                {
                  "type": "text",
                  "value": "You can start lists with extra indentation…\n…and continue them without indentation.",
                  "position": {
                    "start": {
                      "line": 3,
                      "column": 7,
                      "offset": 75
                    },
                    "end": {
                      "line": 4,
                      "column": 40,
                      "offset": 158
                    }
                  }
                }
              ],
              "position": {
                "start": {
                  "line": 3,
                  "column": 7,
                  "offset": 75
                },
                "end": {
                  "line": 4,
                  "column": 40,
                  "offset": 158
                }
              }
            }
          ],
          "position": {
            "start": {
              "line": 3,
              "column": 5,
              "offset": 73
            },
            "end": {
              "line": 4,
              "column": 40,
              "offset": 158
            }
          }
        },
        {
          "type": "listItem",
          "spread": false,
          "checked": null,
          "children": [
            {
              "type": "paragraph",
              "children": [
                {
                  "type": "text",
                  "value": "Part of the current list",
                  "position": {
                    "start": {
                      "line": 5,
                      "column": 8,
                      "offset": 166
                    },
                    "end": {
                      "line": 5,
                      "column": 32,
                      "offset": 190
                    }
                  }
                }
              ],
              "position": {
                "start": {
                  "line": 5,
                  "column": 8,
                  "offset": 166
                },
                "end": {
                  "line": 5,
                  "column": 32,
                  "offset": 190
                }
              }
            },
            {
              "type": "list",
              "ordered": false,
              "start": null,
              "spread": false,
              "children": [
                {
                  "type": "listItem",
                  "spread": false,
                  "checked": null,
                  "children": [
                    {
                      "type": "paragraph",
                      "children": [
                        {
                          "type": "text",
                          "value": "Nested!",
                          "position": {
                            "start": {
                              "line": 6,
                              "column": 10,
                              "offset": 200
                            },
                            "end": {
                              "line": 6,
                              "column": 17,
                              "offset": 207
                            }
                          }
                        }
                      ],
                      "position": {
                        "start": {
                          "line": 6,
                          "column": 10,
                          "offset": 200
                        },
                        "end": {
                          "line": 6,
                          "column": 17,
                          "offset": 207
                        }
                      }
                    }
                  ],
                  "position": {
                    "start": {
                      "line": 6,
                      "column": 8,
                      "offset": 198
                    },
                    "end": {
                      "line": 6,
                      "column": 17,
                      "offset": 207
                    }
                  }
                }
              ],
              "position": {
                "start": {
                  "line": 6,
                  "column": 8,
                  "offset": 198
                },
                "end": {
                  "line": 6,
                  "column": 17,
                  "offset": 207
                }
              }
            }
          ],
          "position": {
            "start": {
              "line": 5,
              "column": 6,
              "offset": 164
            },
            "end": {
              "line": 6,
              "column": 17,
              "offset": 207
            }
          }
        },
        {
          "type": "listItem",
          "spread": false,
          "checked": null,
          "children": [
            {
              "type": "paragraph",
              "children": [
                {
                  "type": "text",
                  "value": "Or more:",
                  "position": {
                    "start": {
                      "line": 7,
                      "column": 7,
                      "offset": 214
                    },
                    "end": {
                      "line": 7,
                      "column": 15,
                      "offset": 222
                    }
                  }
                }
              ],
              "position": {
                "start": {
                  "line": 7,
                  "column": 7,
                  "offset": 214
                },
                "end": {
                  "line": 7,
                  "column": 15,
                  "offset": 222
                }
              }
            },
            {
              "type": "list",
              "ordered": false,
              "start": null,
              "spread": false,
              "children": [
                {
                  "type": "listItem",
                  "spread": false,
                  "checked": null,
                  "children": [
                    {
                      "type": "paragraph",
                      "children": [
                        {
                          "type": "text",
                          "value": "Also nested!",
                          "position": {
                            "start": {
                              "line": 8,
                              "column": 12,
                              "offset": 234
                            },
                            "end": {
                              "line": 8,
                              "column": 24,
                              "offset": 246
                            }
                          }
                        }
                      ],
                      "position": {
                        "start": {
                          "line": 8,
                          "column": 12,
                          "offset": 234
                        },
                        "end": {
                          "line": 8,
                          "column": 24,
                          "offset": 246
                        }
                      }
                    }
                  ],
                  "position": {
                    "start": {
                      "line": 8,
                      "column": 10,
                      "offset": 232
                    },
                    "end": {
                      "line": 8,
                      "column": 24,
                      "offset": 246
                    }
                  }
                }
              ],
              "position": {
                "start": {
                  "line": 8,
                  "column": 10,
                  "offset": 232
                },
                "end": {
                  "line": 8,
                  "column": 24,
                  "offset": 246
                }
              }
            }
          ],
          "position": {
            "start": {
              "line": 7,
              "column": 5,
              "offset": 212
            },
            "end": {
              "line": 8,
              "column": 24,
              "offset": 246
            }
          }
        }
      ],
      "position": {
        "start": {
          "line": 3,
          "column": 5,
          "offset": 73
        },
        "end": {
          "line": 8,
          "column": 24,
          "offset": 246
        }
      }
    }
  ],
  "position": {
    "start": {
      "line": 1,
      "column": 1,
      "offset": 0
    },
    "end": {
      "line": 9,
      "column": 1,
      "offset": 247
    }
  }
}
