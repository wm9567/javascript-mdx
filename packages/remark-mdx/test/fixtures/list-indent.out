As we don’t support indented code, lists can be indented some more.

*   You can start lists with extra indentation…
    …and continue them without indentation.
*   Part of the current list
    *   Nested!
*   Or more:
    *   Also nested!
